deps/wazuh_testing/wazuh_testing/tools/file.py

```diff
@@ -111,3 +117,55 @@ def write_json_file(file_path, data, ensure_ascii=False):
         escaped. If ensure_ascii is false, these characters will be output as-is.
     """
     write_file(file_path, json.dumps(data, indent=4, ensure_ascii=ensure_ascii))
+
+
+def check_file_exist(file_path):
+    return os.path.exists(file_path)
+
+
+def download_file(source_url, dest_path):
+    r = requests.get(source_url, allow_redirects=True)
+    with open(dest_path, 'wb') as f:
+        f.write(r.content)
+
+
+def remove_file(file_path):
+    if check_file_exist(file_path):
+        os.remove(file_path)
+
+
+def validate_json_file(file_path):
+    try:
+        with open(file_path) as f:
+            json.loads(f.read())
+        return True
+    except json.decoder.JSONDecodeError:
+        return False
```

Comment:
```suggestion
    try:
        with open(file_path) as file:
            json.loads(file.read())
        return True
    except json.decoder.JSONDecodeError:
        return False
```

Reply:
Done in 6bf7e6871c48474affeaa53d330e1682285d057f